### ISSUE TITLE
can not view the resulted png image 

### ISSUE THREAD

**Issue by idhamari** [SuperElastix/elastix#61]
OS: Ubuntu 16.04 

I tried to register two png images. I used  (ResultImageFormat "png"). 
The registration works but I can not view the resulted png image. I can  view the result using  (ResultImageFormat "nrrd") in 3D Slicer.

**Comment by idhamari** [comment]
Thanks for your answer. The input images are "Unsigned Char" I tried to use (ResultImagePixelType "Unsigned Char") but the output image size is 0 byte. I also converted the input images to double and used  (ResultImagePixelType "float") but I got 0 byte output. This is the command I used: 
`elastix -m simple90.png -f simple.png -p pars.txt -out out 
`
And these are the files:
![simple](https://user-images.githubusercontent.com/28802516/38486252-bce64376-3bdc-11e8-8482-1282627501d0.png)
![simple90](https://user-images.githubusercontent.com/28802516/38486253-bcfd1b14-3bdc-11e8-8450-7f56bcc763db.png)
[pars.txt](https://github.com/SuperElastix/elastix/files/1888921/pars.txt)
